lib/grandpa/vote_message.go

```diff
@@ -36,6 +38,33 @@ func (s *Service) receiveMessages(ctx context.Context) {
 			logger.Tracef("received vote message %v from %s", msg.msg, msg.from)
 			vm := msg.msg
 
+			switch vm.Message.Stage {
+			case prevote:
+				err := telemetry.GetInstance().SendMessage(
+					telemetry.NewAfgReceivedPrevoteTM(
+						vm.Message.Hash,
+						fmt.Sprintf("%d", vm.Message.Number),
```

Comment:
nit shorter. Same for other calls to `fmt.Sprintf` below

```suggestion
						fmt.Sprint(vm.Message.Number),
```